js/validator.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Created by peterwilkins on 02/12/2015.
+ */
+
+
+
+$(function () {
+
+    $('#target').submit(function (e) {
+        e.preventDefault();
+        // required
+        if (!$('#field1').val()) {
+            $("#err1").alert('close');
+            $('#field1').after(
+                '<div class="alert alert-danger alert-dismissable" id="err1">' +
+                '<button type="button" class="close" ' +
+                'data-dismiss="alert" aria-hidden="true">' +
+                '&times;' +
+                '</button>' +
+                'Required field' +
```

Comment:
the validation message is exactly the same as the field label, maybe you could grab the text from there to save typing it in?